### PR TITLE
STYLE CORE ELEMENTS OF THE PAGE. 

### PR DESCRIPTION
**Pull request by Ntwali-Josue**
### In this pull request, we performed these actions: 

- Add bootstrap links
- Add bootstrap classes
- Remove unused files.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,25 +4,63 @@
...
+    <title>Manage Books</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We"
+      crossorigin="anonymous"
+    />
     <script src="./js/main.js" defer></script>
   </head>
   <body>
```
Comment (rloterh) [inline index.html:17]: - [x] Your design interface does not match  the provided [wireframe template](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m3_style_core_elements.md). Comparing your design with the wireframe template, can can observe that:
- The heading title `All awesome books` appears at the center of the page
- The `remove buttons` appear at the extreme right of each table row
- A form comes with the heading title `Add a new book`
- A narrower `form inputs`
- An `add button` to the right of the form

 We actually want our designs to mimic the provided wireframe as much as possible. Kindly ensure to improve on the highlighted points in an attempt to satisfy a client's demands.

**Your design:**
image


**The original wireframe design:**
image

File: js/main.js [hunk 2]
```diff
@@ -1,60 +1,120 @@
...
-  const renderTitle = templateClone.querySelector(
-    '[data-list-item-book-title]',
-  );
-  const renderAuthor = templateClone.querySelector(
-    '[data-list-item-book-author]',
-  );
-  renderTitle.innerText = bookInfo.title;
-  renderAuthor.innerText = bookInfo.author;
-  booksListContainer.appendChild(templateClone);
+// Book Class: Represents a book
```
Comment (rloterh) [inline js/main.js]: - [x] OPTIONAL
Kindly endeavor to remove **all** comments in your code hosted on the repository so as to enhance a cleaner code.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Team, 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Good Points 👍  
- App logic is working as intended
- No linter errors


## Required Changes ♻️
- [x] _Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [x] _Kindly check the comments marked [OPTIONAL] under the review._

You can also consider:
- [OPTIONAL] 
Kindly endeavor to Update your ReadMe's content to correspond to the current project you are working on. This will make our ReadMe, and project as a whole look more professional


Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mimipeshy — APPROVED** [review]
## Hi @Team 👋🏾 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- As the previous TSE asked, kindly update your README live demo link to reflect the latest changes you made
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Comment by DanSam5K** [comment]
## Hi @mimipeshy 👍 ,
- Thank you for your feedback we have considered the optional changes you suggested and have acted accordingly
👏 👏 👏

**Pull request merged**